tuf/src/repository/ephemeral.rs

```diff
@@ -358,4 +445,85 @@ mod test {
             );
         })
     }
+
+    #[test]
+    fn ephemeral_repo_batch_commit_fails_with_metadata_conflicts() {
```

Comment:
possibly also test that two concurrent batch updates will cause one to fail

Reply:
Good idea, and added some extra impls we can now cover.